File: gestao-spring/src/main/java/com/erp/gestao/infra/model/ViaCepResponse.java
package com.erp.gestao.infra.model;

import lombok.Data;

@Data
public class ViaCepResponse {
    private String cep;

    private String logradouro;

    private String complemento;

    private String bairro;

    private String localidade;

    private String uf;

    private String estado;


    private String regiao;


    private String ibge;

    private String ddd;

    private Boolean erro;
}

File: gestao-angular/src/app/infra/model/via.cep.response.ts
import {Cidade} from "../../shared/model/cidade";

export class ViaCepResponse {
   cep: string;

   logradouro: string;

   complemento: string;

   bairro: string;

   localidade: string;

   uf: string;

   estado: string;

   regiao: string;

   cidade: Cidade;

   ibge: string;

   ddd: string;

   erro: boolean;
}
